app/assets/stylesheets/navbar.scss

```
@@ -10,16 +10,18 @@
 
   .navbar-nav .nav-link {
     /* Change the link color */
-    color: rgba(255,255,255,.55);
+    color: white;
```

Maybe it's just me but pure white seems a tad harsh to my eye. This is very slightly softer whilst still passing the tests:
```suggestion
    color: rgba(255, 255, 255, 0.95);
```